packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -152,3 +186,78 @@ async def publish(self, exchange_name: str, message: Message) -> None:
                 aio_pika.Message(message.encode()),
                 routing_key="",
             )
+
+    async def rpc_request(
+        self,
+        namespace: RPCNamespace,
+        method_name: str,
+        *,
+        timeout: Optional[PositiveInt] = 5,
+        **kwargs: dict[str, Any],
+    ) -> Any:
+        """
+        Call a remote registered `handler` by providing it's `namespace`, `method_name`
+        and list of expected arguments.
+
+        NOTE: `namespace` should always be composed via `get_namespace`
```

Comment:
you might expand the doc string with what is possibly raised

Reply:
and also what are the kwargs... method parameters.

Reply:
aand question, they need to be always keywords or also args would work?

Reply:
> aand question, they need to be always keywords or also args would work?

Yes, there is only a kwargs option in the interface